README.md

```diff
@@ -2,17 +2,24 @@
 
 This module implements the [`rdk:component:board` API](https://docs.viam.com/components/board/#api) and [`rdk:component:servo` API](https://docs.viam.com/components/servo/#api)
 
-The following models are provided in this module:
+This module provides the following models to access GPIO functionality (input, output, PWM, power, serial interfaces, etc.):
 
-* `viam:raspberry-pi:rpi5` - Configure a Raspberry Pi 5 board to access GPIO functionality: input, output, PWM, power, serial interfaces, etc.
+* `viam:raspberry-pi:rpi5` - Configure a Raspberry Pi 5 board 
 * `viam:raspberry-pi:rpi4` - Configure a Raspberry Pi 4 board
 * `viam:raspberry-pi:rpi3` - Configure a Raspberry Pi 3 board
 * `viam:raspberry-pi:rpi2` - Configure a Raspberry Pi 2 board
 * `viam:raspberry-pi:rpi1` - Configure a Raspberry Pi 1 board
 * `viam:raspberry-pi:rpi0` - Configure a Raspberry Pi 0 board
 * `viam:raspberry-pi:rpi0_2` - Configure a Raspberry Pi 0 2 W board
+
+This module also provides a servo model:
+
 * `viam:raspberry-pi:rpi-servo` - Configure a servo controlled by the GPIO pins on the board. Note this model is not supported on the rpi5 board.
 
+Note that Raspberry Pi 0, 1, 2 and 3 are supported as GPIO peripherals but are not capable of running `viam-server` themselves.
```

Comment:
mmm, the 3 does have arm64 support - so you could theoretically run viam server arm64 on there and this module. I believe you can get an armv7l on the 2B+, and we have a 32-bit distribution of this module.

See here for an explanation of whats' going on: https://www.raspberrypi.com/news/raspberry-pi-os-64-bit/

Rather than go into this in the module - I'm hoping we can organically get issues and asks from users so we can 1. get an armv6l image a priority 2. put some time on more distributions of this module when we can - we do have a 32 bit image (probably buggy, but hey)